Auto/Tactic.lean

```diff
@@ -284,13 +284,18 @@ def queryTPTP (exportFacts : Array REntry) : LamReif.ReifM (Array Embedding.Lam.
 
 open Embedding.Lam in
 def querySMT (exportFacts : Array REntry) (exportInds : Array MutualIndInfo) : LamReif.ReifM (Option Expr) := do
-  let lamVarTy := (← LamReif.getVarVal).map Prod.snd
+  -- GEORGE: do we need to pass more of `LamReif:State` to `lamFOL2SMT`?
+  let lamVarTy ← LamReif.getVarVal
+  trace[auto.lamReif.printValuation] "lamVarTy: {lamVarTy}"
   let lamEVarTy ← LamReif.getLamEVarTy
+  trace[auto.lamReif.printValuation] "lamEVarTy: {lamEVarTy}"
```

Comment:
One thing I'm confused about is also that `printValuation` (in `LamReif.lean`) doesn't print any `Etom` lines (suggesting `lamEVarTy` is empty), but this prints `lamEVarTy: [#0, #1, #1, #2]` on the `Names.lean` example.

Is `printValuation` called too early?

Reply:
``etom``s are created by skolemization. Maybe we can use names like ``sk_<num>``.

Reply:
Some automated theorem provers use ``sk`` for skolems, but other names are also ok.